database/migrations/2023_09_15_114234_add_columns_to_users_table.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddColumnsToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('username')->after('name');
+            $table->enum('status', ['active', 'inactive'])->after('password')->default('active');
+            $table->foreignId('role_id')->nullable()->constrained()->after('status');
+            $table->string('address')->after('role_id')->nullable();
+            $table->string('phone_number')->after('address')->nullable();
+            $table->string('image')->after('phone_number')->nullable();
```

Comment:
I suggest to use longText text to image. Because we will be using base 64 to store image as string to database. It should be like this.

$table->longText('image')->after('phone_number')->nullable();

Reply:
I fixed this part